tests/x11regressions/remote_desktop/onetime_vncsession_multilogin_failed.pm

```diff
@@ -0,0 +1,84 @@
+# Copyright (C) 2017 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Remote Login: One-time VNC Session failed due to a previous graphical session
+# Maintainer: Chingkai <qkzhu@suse.com>
+# Tags: tc#1586208
+
+use strict;
+use base 'basetest';
+use testapi;
+use lockapi;
+use utils;
+
+sub start_vncviewer {
+    x11_start_program 'vncviewer 10.0.2.1:1 -Fullscreen';
+    assert_screen [qw(displaymanager vnc_certificate_warning)];
+    if (match_has_tag 'vnc_certificate_warning') {
+        send_key 'ret';
+        assert_screen [qw(displaymanager vnc_certificate_warning-2)];
+        if (match_has_tag 'vnc_certificate_warning-2') {
+            send_key 'ret';
+        }
+    }
+}
+
+sub run() {
+    my $self = shift;
+
+    #wait for supportserver if not yet ready
+    mutex_lock 'dhcp';
+    mutex_unlock 'dhcp';
+    mutex_lock 'xvnc';
+
+    # Make sure the client gets the IP address
+    x11_start_program 'xterm';
+    assert_screen 'xterm';
+    become_root;
+    assert_script_run 'dhclient';
+    type_string "exit\n";
+    send_key 'alt-f4';
```

Comment:
that should be guarded by `wait_screen_change { }` as just afterwards you want to use `start_vncviewer` which assumes the system to be ready